src/main/java/com/minecolonies/client/gui/WindowHireWorker.java

```diff
@@ -171,8 +172,15 @@ public void onButtonClicked(Button button)
     {
         if (button.getID().equals(BUTTON_DONE))
         {
-            Label id = (Label)button.getParent().getChildren().get(CITIZEN_ID_LABEL_POSITION);
-            MineColonies.getNetwork().sendToServer(new HireFireMessage(this.building,true, Integer.parseInt(id.getLabelText())));
+            Label idLabel = (Label)button.getParent().getChildren().get(CITIZEN_ID_LABEL_POSITION);
+            int id = Integer.parseInt(idLabel.getLabelText());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'id' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)